client/src/components/Shared/SearchBar/SearchBar.tsx

```diff
@@ -1,130 +1,134 @@
-          <Option value="categories">Gene Categories</Option>
-        </Select>
-      </div>
-      <div className="search-input">
-        <Form.Item>
-          <Select 
-            allowClear
-            size="large" 
-            placeholder="" 
-            mode="tags"
-            tokenSeparators={[',', ' ']}
-            options={options}
-            onInputKeyDown={onKeyDown}
-            value={state.searchTerms}
-            onClear={() => state.searchTerms = []}
-            onDeselect={(val: any) => dispatch({type: ActionTypes.DeleteTerm, payload: val})}
-            // onChange={value => setQueryParams(value)}
-            onSearch={value => setInputValue(value)}
-          >
-            {state.searchTerms}
-          </Select>
-        </Form.Item>
+  const handleSearchClick = () => {
+    state.searchTerms = selectedOptions.map(option => option.name)
+    handleSubmit();
+  }
+
+  useEffect(() => {
+    // populate tags from state if nothing entered
+    if (selectedOptions?.length !== 0) {
+      selectedOptions.map(option => {
+        dispatch({type: ActionTypes.AddTerm, payload: option.name})
+      });
+    // populate 
+    } else if (state.searchTerms?.length > 0 && selectedOptions?.length === 0) {
+      setSelectedOptions(state.searchTerms.map(option => {return {name: option}}))
+    }
+  }, [selectedOptions]);
 
-          <div className="search-filters">
-            <Popover 
-              content={content} 
-              trigger="click" 
-              open={showFilters} 
-              onOpenChange={open => setShowFilters(open)} 
-            >
-              {/* TODO: Reintroduce later
-              <FilterOutlined
-                style={{ fontSize: '150%', cursor: 'pointer'}}
-              /> */}
-            </Popover>
-          </div>
-        </div>
-      </div>
-  </div>
-  )
+  return (
+    <>
+    <Box>
+    <Box display='flex'>
+      <Box>
+      <Select value={state.interactionMode || searchType} defaultValue={state.interactionMode || 'gene'} onChange={handleChange} classes={{select: 'search-type-select'}}>
+        <MenuItem value='gene'>Interactions by Gene</MenuItem>
+        <MenuItem value='drug'>Interactions by Drug</MenuItem>
+        <MenuItem value='categories'>Gene Categories</MenuItem>
+      </Select>
+      </Box>
+      <Box display='block' ml={1}>
+      <Autocomplete
+        multiple
+        filterSelectedOptions
+        id="tags-standard"
+        options={isValidSearch ? [{name: typedSearchTerm}, ...autocompleteOptions] : autocompleteOptions}
+        getOptionLabel={(option: any) => option.name}
+        renderInput={(params) => (
+          <Box width={650}>
+          <TextField
+            {...params}
+            variant="standard"
+            label="Search Terms"
+          />
+          </Box>
+        )}
+        value={selectedOptions}
+        onInputChange={handleInputChange}
+        onChange={handleAutocompleteChange}
+      />
```

Comment:
```suggestion
        onInputChange={handleInputChange}
        onChange={handleAutocompleteChange}
        isOptionEqualToValue={(option, value) => option.name === value.name}
      />
```

@katiestahl this silences the warning